server/resources/resources_suite_test.go

```diff
@@ -42,6 +42,8 @@ const (
 	adminTenantID     = "fc394f2ab2df4114bde39905f800dc57"
 	memberTenantID    = "fc394f2ab2df4114bde39905f800dc57"
```

Comment:
adminTenantID is the same as memberTenantID. my strong guess is that the values does not really matter, so could we use more human-friendly consts here? I propose just `adminTenantID = "adminTenantID"`. Rationale: if we see a human friendly name in a test error log it'll be easier to debug